internal/highway/x/matrix/config.go

```diff
@@ -42,16 +49,64 @@ func defaultConfig(host host.SonrHost) (config.Dendrite, error) {
 	return cfg, nil
 }
 
-func (p *MatrixProtocol) Init() {
+func (p *MatrixProtocol) setupMonolith() error {
```

Comment:
function should be renamed to better sugest standing up of matrix protocol server